spec/unit/jobs/pollable_job_wrapper_spec.rb

```diff
@@ -119,6 +123,56 @@ module VCAP::CloudController::Jobs
       end
     end
 
+    describe 'error' do
+      let(:job) { double(job_name_in_configuration: 'my-job', max_attempts: 2, perform: nil, guid: '15') }
+      let!(:actual_pollable_job) { VCAP::CloudController::PollableJobModel.create(delayed_job_guid: job.guid) }
+
+      context 'with a big backtrace' do
+        it 'culls it down' do
+          raise BigException.new
+        rescue BigException => exception
```

Comment:
are you raising an error just to get an instance of an exception?  I think you can do this more explicitly like this:
```
exception = BigException.new
exception.set_backtrace(long_string)
```

Reply:
I learn something with every tweak. This also removes the dependency on different path lengths between travis and other environments, so no need to test for a range of expected array sizes -- the final exception size will always be 8 (start with 32 * 1000-char backtrace + the rest of the body, about 200 chars. Divide the backtrace until the full size < 16000 chars. Meaning we can stop with an 8-line backtrace.